react-native/react/native/pinentry.wrapper.desktop.js

```diff
@@ -31,8 +37,8 @@ class PinentryWrapper extends Component {
   render () {
     if ('payload' in this.state) {
       return <Pinentry
-        onSubmit={this.onSubmit}
-        onCancel={this.onCancel}
+        onSubmit={(passphrase, features) => this.onSubmit(passphrase, features)}
```

Comment:
i think its better to just use arrows here to be explicit about the scope. before the this was some render button etc